django/db/models/query.py

```diff
@@ -435,11 +435,12 @@ def bulk_create(self, objs, batch_size=None):
         if not objs:
             return objs
         self._for_write = True
-        connection = connections[self.db]
+        db = self.db
```

Comment:
@timgraham I was the one who suggested storing `db` in a locale variable to prevent multiple attribute lookups in the primary assignment loop. Might be overkill as you both pointed out.